Dockerfiles/midstream/run_tests.py

```diff
@@ -42,24 +42,45 @@ def run_validate_operator_bundle(self):
+        result = subprocess.run(exec_cmd, shell=True, capture_output=True)
         return result
 
     def test_for_extract_and_validate_bundle_image(self):
+        
+        self.setUp()
+        global exit_code
+        exit_code = 0
         result = self.run_extract_operator_bundle()
-        self.assertEqual(0,
-                         result.returncode,
-                         "extract-operator-bundle.yml failed")
+        if(result.returncode != 0):
+            print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file extract-operator-bundle.errormessage for more info." % result.returncode)
+            with open("extract-operator-bundle.errormessage", "w") as errorMsg:
+                print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=errorMsg)
+            exit_code = 102
+            return exit_code
         result = self.run_validate_operator_bundle()
-        self.assertEqual(0,
-                         result.returncode,
-                         "validate-operator-bundle.yml failed")
-        self.assertTrue(path.exists("/project/output/validation-rc.txt"))
-        self.assertTrue(path.exists("/project/output/validation-output.txt"))
-        self.assertTrue(path.exists("/project/output/validation-version.txt"))
+        if(result.returncode != 0):
+            print("Ansible playbook validate-operator-bundle.yml failed with result code: %s , see the file validate-operator-bundle.errormessage for more info." % result.returncode)
+            with open("validate-operator-bundle.errormessage", "w") as errorMsg:
+                print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=errorMsg)
+            exit_code = 103
+            return exit_code
+        assert (path.exists("/project/output/validation-rc.txt"))
+        assert (path.exists("/project/output/validation-output.txt"))
+        assert (path.exists("/project/output/validation-version.txt"))
         with open('/project/output/validation-rc.txt', 'r') as reader:
             rc = reader.read()
-            self.assertEqual(0, int(rc), "validation of bundle image failed")
+            if(int(rc) != 0):
+                print("Image bundle validation failed with result code: %s , see /project/output/validation-output.txt file for more info." % int(rc))
+                exit_code = 70
+                return exit_code
+        return 0
 
 if __name__ == '__main__':
-    unittest.main()
+    resultcodes = [0,102,103,70]
+    runTest = RunMidstreamCVPTests()
+    with open("tests_result.log",'w') as f:
+        if (runTest.test_for_extract_and_validate_bundle_image() not in resultcodes):
+            exit_code = 1
+            print("Error occured during unit tests, please see tests_result.errormessage file for more info.")
+            os.rename(r'tests_result.log',r'tests_result.errormessage')
```

Comment:
The requirements in the story says:
```
.errormessage 
Should be place in $PWD (where the entrypoint script is located).
```
The requirements does not say anything about tests_result.errormessage

Reply:
Changed to .errormessage

Reply:
@jsztuka I still cannot see the change.